src/lib/components/NavbarComponent.svelte

```diff
@@ -4,7 +4,7 @@
 
 <nav>
     <ul
-        class="flex flex-row items-center justify-between gap-4 rounded-xl bg-warm-white py-1 pl-8 pr-1"
+        class="m-2 flex flex-row items-center justify-between gap-4 rounded-xl bg-warm-white py-1 pl-8 pr-1"
```

Comment:
Let's avoid placing the `margin` at the component-level. Ideally, this is something we put at the page-level or the layout-level to give maximum flexibility to whoever is gonna use our component. For instance, it would be unfortunate if importing the component forced us to include the `m-2` margin.